rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -2,13 +2,13 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_maintain
-%global confdir foreman_maintain
+%global package_name foreman-maintain
```

Comment:
Especially when the gem does follow the ```foreman_<name>``` convention. Using the same convention in the package name makes it very easy to install in puppet-foreman.